generic_config_updater/change_applier.py

```diff
@@ -32,8 +33,8 @@ def log_error(m):
     logger.log(logger.LOG_PRIORITY_ERROR, m, print_to_console)
 
 
-def get_config_db():
-    config_db = ConfigDBConnector()
+def get_config_db(namespace=multi_asic.DEFAULT_NAMESPACE):
```

Comment:
we have this function `connect_config_db_for_ns` here https://github.com/sonic-net/sonic-buildimage/blob/725f62ac7c5d04cc8709d05925aaa7e1a3f292d8/src/sonic-py-common/sonic_py_common/multi_asic.py#L35 can we resue this function?